packages/blockchain/README.md

```diff
@@ -239,9 +239,15 @@ The `Blockchain` class has a public property `events` which contains an `EventEm
 | ------------------------ | ------------------------------------------- |
 | `deletedCanonicalBlocks` | Emitted when blocks are reorged and deleted |
 
-## Developer
+## Debugging
 
-For debugging blockchain control flows the [debug](https://github.com/visionmedia/debug) library is used and can be activated on the CL with `DEBUG=[Logger Selection] node [Your Script to Run].js`.
+For the debugging output to show up, set the `DEBUG` environment variable, e.g. in Linux/Mac OS:
+`export DEBUG=ethjs,*`
+
+`ethjs` **must** be included in the `DEBUG` environment variables to enable **any** logs.
+Additional log selections can be added with a comma separated list (no spaces). Logs with extensions can be enabled with a colon `:`, and `*` can be used to include all extensions.
+
+`DEBUG=ethjs,thislog,thatlog,otherlog,otherlog:sublog,anotherLog:* node myscript.js`
```

Comment:
I think this "thislog, thatlog" is not very helpful, can me make this concrete and use an example from the library itself (also goes for some other docs below)?

It would also be good if all the loggers are just listed here as well (other library as well), even if there are only 2-3, so that users directly get an overview and guidance what to use.

Reply:
The `blockchain`, `verkle`, and `client` packages don't have enough namespaces available to make a concrete example that demonstrates using both specific namespaces and wildcards, so I reused the `statemanager` example for those packages.

Also collated and added the namespaces available in the client package as well since that one was missed, but the blockchain namespaces are fully listed out here, there's just a single one available.

Reply:
Ok, that might be ("packages don't have enough namespaces available") but we DO actually want to document the usage of the specific packages here and it really is not that helpful to users if they find statemanager examples on the blockchain package (and if there is no usage for wildcards for blockchain so be it, then we do not need to document there and can just remove from the docs). Will make this small update and then merge, otherwise this looks good! 🙂

Reply:
Update: ah, just seeing that there is also a blockchain example, ok. If possible we should then optimally rather use some other-lib examples which makes some sense in the documented library context, but for blockchain admittedly this is also difficult, guess there is no other logger (from our dependencies) which can really be used. Anyhow: I'll look a bit over the examples and maybe make some small adjustments (e.g. for client it is more of a "real world" example if one is citing the `devp2p` loggers e.g. there, statemanager is not really a common debug need there, and then I'll merge.